includes/copyiteminc.php

```diff
@@ -461,6 +477,20 @@ function doaftercopy($sourcecid) {
 			}
 		}
 	}
+	//update any assessment ids in categories
+	if (count($categoryassessmenttrack)>0) {
+		foreach ($categoryassessmenttrack as $newqid=>$oldcategoryaid) {
+			//is oldcategoryaid in copied list?
+			if (isset($assessnewid[$oldcategoryaid])) {
+				$query = "UPDATE imas_questions SET category='AID-{$assessnewid[$oldcategoryaid]}' WHERE id='$newqid'";	
+				mysql_query($query) or die("Query failed : $query" . mysql_error());
+			} else if (!$samecourse) { //since that assessment isn't being copied, unclear what category should be
+				$query = "UPDATE imas_assessments SET category=0 WHERE id='$newqid'";

```

Comment:
If the assessment wasn't copied, I assumed category should be 0.  Let me know if that's not the best choice.